entrypoint.sh

```diff
@@ -71,7 +71,10 @@ case "${ACTION}" in
 esac
 
 echo "${OUTPUT}"
-echo ::set-output name=detailed::"${OUTPUT}"
+SINGLE_LINE_OUTPUT="${OUTPUT//'%'/'%25'}"
+SINGLE_LINE_OUTPUT="${SINGLE_LINE_OUTPUT//$'\n'/'%0A'}"
+SINGLE_LINE_OUTPUT="${SINGLE_LINE_OUTPUT//$'\r'/'%0D'}"
```

Comment:
I'm not sure all these single quotes are correct. For example if `OUTPUT=$(echo -e 'hello\nworld')` then the echo prints:

```
::set-output name=detailed::hello'%0A'world
```

Reply:
Tested this in my repo and it works,
I copied it from this article:
https://trstringer.com/github-actions-multiline-strings/

Reply:
Without it, it only prints the first line

Reply:
Ok!